doc/ven_impl/ven_template/README.md

```diff
@@ -0,0 +1,40 @@
+# CNTT Vendor Implementation - VI
+
+## Vendor Name
+Vendor XX
+
+## Target Reference Architecture :
+|   Type          | Release  | Notes
+|--------------------------------|--------------------------------|-------------------------------|
+| OpenStack (RA-1) |  v3.0   |   |
+
+## Target Reference Conformance :
+|   Type          | Release  | Notes
+|--------------------------------|--------------------------------|-------------------------------|
+| OpenStack (RC-1) |  v3.0   |   |
+
+## VI Release Information
```

Comment:
Is it sufficient to get version numbers or do we need more details about the Vendor Implementation

Reply:
more details will be in the cookbook, this is release details of the VI in CNTT. NFVI version that relates to a product will be in the cookbook itself.